lib/validation-stream.js

```diff
@@ -9,6 +9,12 @@ var Duplex = require('readable-stream').Duplex;
 
 var XmlParser = require('./xml-parser');
 
+/** @arg {?string} str Input string to check.
+    @return {!boolean} True if input is undefined, null, or entirely whitespace. */
+function isBlank(str) {
+  return !str || !/\S/.test(str);
```

Comment:
Ah, I totally missed that. The name `str` implies that we expect a string and never `undefined` or `null`. A better name might have been `input`. Additionally, we could make it clearer what the code is trying to achieve by doing something like:

```js
return typeof input === 'string' ? !/\S/.test(str) : false;
```